tools/staticbuild/build_lib_cmake.sh

```diff
@@ -30,7 +30,12 @@ git submodule update --init --recursive || true
 
 # Build libmxnet.so
 rm -rf build; mkdir build; cd build
-cmake -GNinja -C $cmake_config -DCMAKE_PREFIX_PATH=${DEPS_PATH} -DCMAKE_FIND_ROOT_PATH=${DEPS_PATH} ..
+cmake \
+    -GNinja \
+    -C $cmake_config \
+    -DCMAKE_PREFIX_PATH=${DEPS_PATH} \
+    -DCMAKE_OSX_DEPLOYMENT_TARGET=10.13 \
```

Comment:
Duplicate?

Reply:
should have no harm.

Reply:
If there's no benefit, there's no need to add the code. It will just serve to confuse readers